jujugui/static/gui/src/app/components/deployment-flow/signup/signup.js

```diff
@@ -0,0 +1,144 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2016 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+YUI.add('deployment-signup', function() {
+
+  juju.components.DeploymentSignup = React.createClass({
+    propTypes: {
+      changeState: React.PropTypes.func.isRequired,
+      exportEnvironmentFile: React.PropTypes.func.isRequired,
+      modelName: React.PropTypes.string.isRequired
+    },
+
+    /**
+      Handle navigating to the deployment flow.
+
+      @method _displayFlow
+    */
+    _displayFlow: function() {
+      this.props.changeState({
+        sectionC: {
+          component: 'deploy',
+          metadata: {
+            activeComponent: 'flow'
+          }
+        }
+      });
+    },
+
+    /**
+      Handle the email registration.
+
+      @method _handleSignup
+    */
+    _handleSignup: function() {
+      console.error('Not implemented.');
+    },
+
+    render: function() {
+      return (
+        <juju.components.DeploymentPanel
+          changeState={this.props.changeState}
+          title={this.props.modelName}>
+          <div className="deployment-signup">
+            <div className="six-col">
+              <h2>Install Juju to deploy locally</h2>
+              <p>
+                Doing so provides on LXD, which allows you to recreate the
+                production deployment environment on your own machine. This
+                minimises portability issues when deploying to a public cloud,
+                OpenStack or bare metal.
+              </p>
+              <p>To deploy locally:</p>
+              <ol>
+                <li>
+                  <juju.components.GenericButton
+                    action={this.props.exportEnvironmentFile}
```

Comment:
This is a limitation with the current model export (it can only currently export committed changes).